web-console/src/dialogs/lookup-edit-dialog/lookup-edit-dialog.tsx

```diff
@@ -93,8 +194,365 @@ export const LookupEditDialog = React.memo(function LookupEditDialog(props: Look
+      info: 'Defines the connectURI value on the The connector config to used',
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.connectorConfig.user',
+      type: 'string',
+      label: 'User',
+      info: 'Defines the user too be used by the connector config',
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.connectorConfig.password',
+      type: 'string',
+      label: 'Password',
+      info: 'Defines the password too be used by the connector config',
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.table',
+      type: 'string',
+      label: 'Table',
+      placeholder: 'some_lookup_table',
+      info: (
+        <p>
+          The table which contains the key value pairs. This will become the table value in the SQL
+          query:
+          <br />
+          SELECT keyColumn, valueColumn, tsColumn? FROM namespace.<strong>table</strong> WHERE
+          filter
+        </p>
+      ),
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.keyColumn',
+      type: 'string',
+      label: 'Key column',
+      placeholder: 'my_key_value',
+      info: (
+        <p>
+          The column in the table which contains the keys. This will become the keyColumn value in
+          the SQL query:
+          <br />
+          SELECT <strong>keyColumn</strong>, valueColumn, tsColumn? FROM namespace.table WHERE
+          filter
+        </p>
+      ),
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.valueColumn',
+      type: 'string',
+      label: 'Value column',
+      placeholder: 'my_column_value',
+      info: (
+        <p>
+          The column in table which contains the values. This will become the valueColumn value in
+          the SQL query:
+          <br />
+          SELECT keyColumn, <strong>valueColumn</strong>, tsColumn? FROM namespace.table WHERE
+          filter
+        </p>
+      ),
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.filter',
+      type: 'string',
+      label: 'Filter',
```

Comment:
add a placeholder of `'(optional)'` that is the convention that I've been using in several places.